app/jobs/identities_backfill_job.rb

```diff
@@ -0,0 +1,58 @@
+class IdentitiesBackfillJob < ApplicationJob
+  # This is a short-term solution to backfill data requiring a table scan.
+  # This job can be deleted once it's done.
+
+  queue_as :low
```

Comment:
Was this appropriate? My thinking is that this isn't too time-sensitive.

Reply:
it's probably fine, I think other backill jobs have been `:long_running` (which also won't page if they run for too long)